src/ape_ethereum/ecosystem.py

```diff
@@ -136,7 +116,7 @@ def encode_address(cls, address: AddressType) -> RawAddress:
         return str(address)
 
     def serialize_transaction(self, transaction: TransactionAPI) -> bytes:
-        return transaction.serialize_transaction()
+        return self.serialize_transaction(transaction)
```

Comment:
Should be just moving this field over:

https://github.com/ApeWorX/ape/blob/3f154b239a783fc4659ba70bd489b2a580002379/src/ape_ethereum/transactions.py#L43